multinode-demo/replicator.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
```

Comment:
This feels like it's gonna be a maintenance PITA.  How about cargo culting `multinode-demo/bootstrap-leader.sh` but with a `--replicator` argument instead of `--bootstrap-leader`

Reply:
Sure. I was trying to keep it clean but ended up with a lot of copy pasta....